alacritty_terminal/src/util.rs

```diff
@@ -94,6 +94,15 @@ where
         .map(|_| ())
 }
 
+// Converts the string slice into a Windows-standard representation for "W"-
+// suffixed function variants, which accept UTF-16 encoded string values.
+#[cfg(windows)]
+pub fn win32_string<S: AsRef<OsStr> + ?Sized>(value: &S) -> Vec<u16> {
```

Comment:
I'm generally not a fan of `util` modules as they easily grow to contain all sorts of random functions. Is there a better home for this? (Moved from `panic.rs` so that `tty` can use it too.)

Reply:
This is something I've been trying to figure out as well, hoping to corral things prior to a `winpty` feature gate in order to cut down on the number of `#[cfg(...)]`'s that will demand. I'd suggest something like `sys/$OS` mod + `impl`, but that might require a big bang development cycle to even sell.

Reply:
I think even `tty/windows/mod.rs` would be a better place than `util.rs`. That's the only real place where we have a big difference between Windows and other OSs anyways.

Though as usual, I think the imports should be moved to the top of the file.